### ISSUE TITLE
Including object properties

### ISSUE THREAD

**Issue by LeilaFattel** [ontodev/robot#1177]
I have been using Robot to create a subset from the gene ontology. I have been able to use the Filter command to create that subset just like the example below (from https://robot.obolibrary.org/filter):
```
robot filter --input uberon_module.owl \
  --prefix "core: http://purl.obolibrary.org/obo/uberon/core#" \
  --select "oboInOwl:inSubset=core:uberon_slim" \
  --select annotations \
  --signature true \
  --output results/uberon_slim.owl
```

However, I would also like to keep the object properties as well, specifically the relationships between the terms (like "part of", "regulates"...). How can one do that? Thank you in advance.

**Comment by LeilaFattel** [comment]
Thank you for the quick reply! I created a term-file and tried the extract subset method and while it is giving me the right terms in the output, it is still not including the object properties (so I am getting a very similar file to the filter method). When I tried the STAR and BOT methods with --intermediates none, although I am getting the relations included this time, I am getting a bigger number of GO terms than I want in my subset. I do not want these "extra" terms to appear at all, just like in any goslim subset file. Any advice?

Examples of what I ran:

`robot extract --method subset --input file.owl --term-file subset_query.txt --output results/subset.owl`

`robot extract --method STAR --input file.owl --term-file subset_query.txt --intermediates none --output results/subset.owl`

`robot extract --method BOT --input file.owl --term-file subset_query.txt --intermediates none --output results/subset.owl`
